.github/workflows/build.yml

```diff
@@ -138,7 +138,7 @@ jobs:
           ctest -L main --verbose --timeout 900
 
   macOS-latest-cmake-arm64-webgpu:
-    runs-on: latest
```

Comment:
Is this actually a self-hosted runner (`latest`) that` is used in this job? 

Could someone bring me up to speed on why we could not use `macos-latest` instead which feels like it might be more reliable?

Reply:
I don't think it is a self-hosted runner. If `macos-latest` runs, don't think there is a problem to use it. cc @reeselevine

Reply:
The job was picked up but there was a test failure which seems to be the same issue as the other macOS-latest-cmake-arm64 job fails on. So perhaps this could be merged just the same as it looks like it at least enabled the job to run.